server/webapp/WEB-INF/vm/shared/_header.vm

```diff
@@ -19,7 +19,7 @@
 <head>
     <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
 
-    <link href="$req.getContextPath()/$concatenatedCruiseIconFilePath?#include("admin/admin_version.txt.vm")" rel="shortcut icon" />
+    <link href="$req.getContextPath()/$concatenatedCruiseIconFilePath?${currentGoCDVersion.fullVersion()}" rel="shortcut icon" />
```

Comment:
Intentional to bust cache :) Rails does this nicely for us by adding a checksum of the resource in the URL.